filebeat/module/iis/w3svc/manifest.yml

```diff
@@ -0,0 +1,19 @@
+module_version: 1.0
+
+var:
+  - name: paths
```

Comment:
@tsg Could you take a look to confirm the log definitions for windows a module that only supports windows are as expected?